src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,378 @@
+for the purposes of a phishing attack.
+
+Current work on the PayID Protocol and its extensions may prove useful in mitigating these risks. 
+
+## Access Control
+As with all web resources, access to the PayID Discovery resource could
+require authentication. See section 6 of [RFC7033][] for Access Control
+considerations.
+
+Furthermore, it is RECOMMENDED that PayID Discovery Servers only expose PayID Easy Checkout URLs
+which resolve to a protected resource.  
+
+# IANA Considerations
+  ## New Link Relation Types
+  This document defines the following Link relation type per [RFC7033][].
+  See section 3 for examples of each type of Link.
+  
+  ### PayID Easy Checkout URL
+  
+    * Relation Type ('rel'): `https://payid.org/ns/payid-easy-checkout-uri/1.0`
+    * Media Type: `application/jrd+json`
+    * Description: PayID Easy Checkout URL, version 1.0
+
+# Acknowledgments
+
+# Appendix
+
+## Motivation
+The PayID Easy Checkout Protocol aims to enable a consistent user experience for payers paying for goods
+or services by standardizing the interaction between merchants/non-profits and customer/donor wallets.
+Given the ability to assign arbitrary metadata to a PayID as defined in [PayID-Discovery][], there is an opportunity
+to standardize the set of interactions between merchant and payer, specifically the process by which a merchant
+directs a payer to their digital wallet to complete a payment.
+We believe this protocol will enable an improved paying experience by reducing the number
+of steps a payer must take to complete a transaction.
+
+PayID Easy Checkout also limits the engineering effort needed to implement the protocol. 
+Clients wishing to adopt this pattern should only need to implement UI-level changes in order to make the flow function 
+as intended, which may aid in expanding overall adoption, further enhancing the protocol's user experience benefits. 
+
+### Design Goals
+
+#### Minimal effort for the Payer
+
+In order for a payer to checkout using the PayID Easy Checkout protocol, the payer only needs to provide a merchant
+with their PayID Easy Checkout enabled PayID.
+
+#### No New Server-Side Software
+
+Apart from a PayID Discovery compliant PayID Discovery Server, The PayID Easy Checkout Protocol does not require server-side 
+software to be run by either the payer or merchant for a payment. The PayID Discovery Server is capable of providing details 
+of where to send the payer via the PayID Discovery Protocol. Assuming the wallet used by the payer has implemented 
+support in their UI for the PayID Easy Checkout Protocol, the payer can be redirected to their wallet 
+to complete their transaction.
+
+## Example Usage
+This section shows a non-normative example of PayID Easy Checkout between a hypothetical merchant (recipient) and payer. The merchant
+accepts payments using the PayID pay$merchant.com, and the payer controls the PayID alice$wallet.com.
```

Comment:
```suggestion
accepts payments using the PayID pay$merchant.example.com, and the payer controls the PayID alice$wallet.example.com.
```

In examples, we should probably use URLs that conform to [RFC-6761](https://tools.ietf.org/html/rfc6761#section-6.5) to prevent accidental usage.